### PR TITLE
[16.0][MIG] coupon_chatter: Migration to version 16.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT44316

@pilarvargas-tecnativa @Yadier-Tecnativa please review!

### CODE REVIEW

File: coupon_chatter/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+To use this module, you need to:
+
+#. Go to either *Sales > Products > Promotion Programs* or *Sales > Products > Coupon
```
Comment (pilarvargas-tecnativa) [inline coupon_chatter/readme/USAGE.rst]: These instructions for use have changed:
image

Comment (stefan-tecnativa) [reply coupon_chatter/readme/USAGE.rst]: Yes, that's true. Thanks for the notice.

**Review by pilarvargas-tecnativa — APPROVED** [review]
thanks :)

**Review by Yadier-Tecnativa — APPROVED** [review]
LGTM

**Pull request merged**